org.mixedrealitytoolkit.input/Readers/PinchInputReader.cs

```diff
@@ -145,16 +141,14 @@ protected virtual void OnDisable()
         /// </summary>
         protected virtual void Update()
         {
-            if (m_firstUpdate)
-            {
-                UpdateActionValidCaches();
-                m_firstUpdate = false;                
-            }
+            m_isTrackingStatePolyfilled = trackedPoseDriver.GetIsPolyfillDevicePose();
+            m_isSelectPolyfilled = !selectAction.action.HasAnyControls();
```

Comment:
Do we want to force needing a selectionAction and selectActionValue.  If the app wants to rely 100% on polyfill, these are necessarily required.   

I would prefer to check for null here...

Reply:
> Do we want to force needing a selectionAction and selectActionValue

Good question! The previous implementation did force needing these actions, but I can potentially see the use-case where an app might prefer the polyfill logic. I can null check the serialized `InputActionProperty`s here too, to account for that.

> I would prefer to check for null here...

Null-checking the action itself happens inside the `HasAnyControls()` extension method:

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/9f1831d385880a403be4693c65331664afe5ca38/org.mixedrealitytoolkit.input/Utilities/InputActionExtensions.cs#L17-L20